boards/imix/src/io.rs

```diff
@@ -39,7 +39,7 @@ impl Write for Writer {
 pub unsafe extern "C" fn panic_fmt(args: Arguments, file: &'static str, line: u32) -> ! {
 
     let writer = &mut WRITER;
-    let _ = writer.write_fmt(format_args!("Kernel panic at {}:{}:\r\n\t\"", file, line));
+    let _ = writer.write_fmt(format_args!("\r\n\nKernel panic at {}:{}:\r\n\t\"", file, line));
```

Comment:
I have the `begin_panic` stuff going in another branch, so it didn't seem worth adding a hack everywhere just to rip it out, but I can add it. Done.